robot/power.py

```diff
@@ -43,6 +56,17 @@ def power_off(self):
         """
         self._send_and_receive({'power': False})
 
+    def set_output(self, output: PowerOutput, value: bool) -> None:
```

Comment:
Hrm on writing the docs I'm realising that this API is actually quite unclear as it stands. It's non-obvious to me what it is that:
```python
r.power_board.set_output(PowerOutput.HIGH_POWER_1, True)
```
actually does. I think it's the combination of `output` and `True`.

Perhaps something like this might be clearer:
```python
r.power_board.disable_output(PowerOutput.HIGH_POWER_1)
r.power_board.enable_output(PowerOutput.HIGH_POWER_1)
```

I'll put up a PR for that.

Reply:
In fact, `power_on_output(...)` and `power_off_output(...)` is better as it follows the existing naming.

Reply:
this is now https://github.com/sourcebots/robot-api/pull/101